docs/adr/0008-interactive-workspace-and-repo-suffixes.md

```diff
@@ -0,0 +1,27 @@
+# 8. Interactive Workspace and Repo suffixes
+Date: 2023-02-22
+
+
+## Status
+Accepted
+
+
+## Context
+Interactive studies fit into our general platform system in that they exist a project under which workspaces, jobs, etc all exist.
+However, we want an easy way to differentiate between an Interactive workspace and "traditional" workspaces.
```

Comment:
Could you say why we "want an easy way to differentiate between an Interactive workspace and "traditional" workspaces"?